deeppavlov/dataset_readers/snips_reader.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
```

Comment:
Shouldn't this copyright be yours? :slightly_smiling_face:

Reply:
Fixed :)